.rebase/replace/code/build/lib/mangle/index.js

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
I guess file has incorrect name: https://github.com/eclipse/che/issues/22428

Reply:
right - renamed in https://github.com/che-incubator/che-code/pull/266